proposals/lip-0055.md

```diff
@@ -78,23 +101,23 @@ Name                          | Type    | Value                  | Description
 ----------------------------- | ------- | ---------------------- | ----------------------------------------------
 `MAX_TRANSACTIONS_SIZE_BYTES` | integer | configurable per chain | The max size of a block transactions in bytes.
 `MAX_ASSET_DATA_SIZE_BYTES`   | integer | configurable per chain | The max size of an assets entry in bytes.
-`SIGNATURE_LENGTH_BYTES`      | integer | 64                     | The length of a Ed25519 signature.
+`SIGNATURE_LENGTH_EDDSA_BYTES`      | integer | 64                     | The length of a Ed25519 signature.
 
 Furthermore, in the following we indicate with `block` be the block under consideration and with `previousBlock` the previous block of the chain. Calling a function `fct` from another module `module` is represented by `module.fct`.
 
 ### Block Processing Stages
 
 <img src="lip-0055/stages.png" width="70%">
 
-*Figure 1: A schematic depiction of the various stages of a block processing. The steps performed in the consensus domain are indicated by blue boxes, while the steps performed in the state-machine domain are indicated by red boxes. Within the state-machine domain, the transactions processing is indicated by purple boxes. The network domain is indicated by green boxes.*
+*Figure 1: A schematic depiction of the various stages of a block processing. The stages performed in the consensus domain are indicated by blue boxes, while the stages performed in the application domain are indicated by red boxes. Within the application domain, the transactions processing is indicated by purple boxes. The network domain is indicated by green boxes.*
```

Comment:
The diagram currently contains the "asset finalization" which is actually not part of the block processing, but only the block generation, I believe.

Reply:
nice catch, fixed